### ISSUE TITLE
Refers to a setting that no longer exists

### ISSUE THREAD

**Issue by Jlouden9** [MicrosoftDocs/memdocs#2715]
Hi,

In the section resolve conflicts for security baselines there are a couple of pointers to "Endpoint Security Configuration" this no longer exists. Might need to update the pointer and confirm that the baselines appear in that section as they don't appear in the device configuration area.

Endpoint security > Security baselines > select a baseline type > Profiles > select a baseline instance > Device Status > **Endpoint security configuration** > settings that show a Conflict or Errors.
Devices > select a device > **Endpoint security configuration** > select a profile or baseline > select a setting from the list of settings that shows a Conflict or Errors.

Regards
John

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1d9e34c6-0221-c998-a264-e74be05cf5a3
* Version Independent ID: cb9d4841-45ba-daba-8acb-df6a43ff30d1
* Content: [Check the success or failure of security baselines in Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/protect/security-baselines-monitor)
* Content Source: [memdocs/intune/protect/security-baselines-monitor.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/protect/security-baselines-monitor.md)
* Service: **microsoft-intune**
* Sub-service: **protect**
* Technology: ****
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by Brenduns** [comment]
@jlouden9   Thanks for this feedback.  Yep, something went off the rails and the left-hand apparently had no idea what the right-hand was up to….  It turns out some time back a [**new Device Configuration report**](https://docs.microsoft.com/mem/intune/fundamentals/reports#device-and-user-check-in-status-operational) was released, which consolidates information not only about device configuration policies, but the endpoint security and security baseline policies as well, which are also device configuration settings.  That this new report also deprecated the view for _Endpoint security configuration_ wasn’t clear. The result was when this view was removed, no one realized we had orphaned content references. Until now.  

I’ve opened a work item to bring this content up to date, direct readers to use the Device Configuration report, and update that reports description to make it clear that this also includes the details from the relevant endpoint security policies and baselines.  

I’ll be closing this issue now that I’ve got that work actively tracked and under way but will still send you an update via this issue once that content is publishing.
